requirements/README.rst

```diff
@@ -12,22 +12,27 @@ directly in the requirements directory.)
 
 .. _OEP-18: https://github.com/openedx/open-edx-proposals/blob/master/oeps/oep-0018-bp-python-dependencies.rst
```

Comment:
Nit: I usually keep these references at the bottom of a subsection, but that is just personal preference.

Reply:
I like keeping them as local as possible unless there are multiple uses of the reference.